.gitignore

```diff
@@ -4,4 +4,4 @@ venv-*/
 .ideaconfig.py
 config.py
 ldap_pw
-mail_pw
\ No newline at end of file
+mail_pw
```

Comment:
isn't mail_pw obsolete now that we have that moved to config.py?

Reply:
Yes, but that's not in master yet.